src/PublicApiAnalyzers/Core/Analyzers/DeclarePublicApiAnalyzer.cs

```diff
@@ -30,6 +32,11 @@ public sealed partial class DeclarePublicApiAnalyzer : DiagnosticAnalyzer
         internal const string InvalidReasonMisplacedNullableEnable = "The '#nullable enable' marker can only appear as the first line in the shipped API file";
         internal const string PublicApiIsShippedPropertyBagKey = "PublicAPIIsShipped";
 
+        /// <summary>
+        /// Boolean option to configure if public API analyzer should bail out silently if public API files are missing.
+        /// </summary>
+        private const string BailOnMissingPublicApiFilesEditorConfigOptionName = "dotnet_public_api_analyzer.require_api_files";
```

Comment:
This should go into `EditorConfigOptionNames`, shouldn't it?

Reply:
I would like to keep `EditorConfigOptionNames` reserved for the core CA rules package. Another reason for not putting it with other options is that public API analyzer is based off a very old Roslyn version (1.2.1) for backwards compatibility, so it cannot compute use `AnalyzerOptionsExtensions` or related APIs for decoding options. This makes it distinct enough to just keep it isolated to this file and package.